src/smc-webapp/course/common.cjsx

```diff
@@ -27,6 +27,7 @@ misc = require('smc-util/misc')
 
 # React libraries
 {React, rclass, rtypes, Actions, ReactDOM}  = require('../smc-react')
+Fragment = React.Fragment
```

Comment:
If we'll use this a lot, it would be nice to just do 
```
{Fragments, React, rclass, rtypes, Actions, ReactDOM}  = require('../smc-react')
```

Reply:
Yeah, I'm going to investigate the actual fragment syntax today. It should work like this:
```
<>
  <Component />
  <AnotherComponent />
</>
```